spec/requests/purchases/tipping_spec.rb

```diff
@@ -143,6 +143,11 @@
 
       fill_in "Tip", with: 5
 
+      expect(page).not_to have_radio_button("0%")
```

Comment:
Spec updated to check percentage options are not visible before and after entering tip value on free carts